kustomize/components/clairpgupgrade/base/clair-pg-old.deployment.yaml

```diff
@@ -57,3 +57,20 @@ spec:
             requests:
               cpu: 500m
               memory: 2Gi
+          lifecycle:
+            postStart:
+              exec:
+                command: ["/bin/sh", "-c", "
+                  until pg_isready; do
+                    echo 'Waiting for PostgreSQL to start...';
+                    sleep 2;
+                  done;
+                  if [ \"\$(cat /proc/sys/crypto/fips_enabled)\" -eq 1 ]; then
```

Comment:
you may want to change this to something like `if [[ -f /proc/sys/crypto/fips_enabled && $(< /proc/sys/crypto/fips_enabled) -eq 0 ]]; then` so it checks for the file

Reply:
My linter says this results in "invalid escape sequence", i'm also getting an error saying it can't inflate this file